aspnet/web-pages/overview/security/using-a-catpcha-to-prevent-automated-programs-bots-from-using-your-aspnet-web-site.md

```diff
@@ -34,11 +34,11 @@ You can help make sure that a user is real person and not a computer program by
 
 ## Adding a ReCaptcha Test
 
-In ASP.NET pages, you can use the `ReCaptcha` helper to render a CAPTCHA test that is based on the ReCaptcha service ([http://recaptcha.net](http://recaptcha.net)). The `ReCaptcha` helper displays an image of two distorted words that users have to enter correctly before the page is validated. The user response is validated by the ReCaptcha.Net service.
+In ASP.NET pages, you can use the `ReCaptcha` helper to render a CAPTCHA test that is based on the [ReCaptcha service](https://www.google.com/recaptcha/about/). The ReCaptcha helper displays an image of two distorted words that users have to enter correctly before the page is validated. The user response is validated by the ReCaptcha.Net service.
 
 ![Screenshot of the Captcha test generated by the ReCaptcha service, showing two distorted words and a text field for user verification.](using-a-catpcha-to-prevent-automated-programs-bots-from-using-your-aspnet-web-site/_static/image1.jpg)
 
-1. Register your website at ReCaptcha.Net ([http://recaptcha.net](http://recaptcha.net)). When you've completed registration, you'll get a public key and a private key.
+1. Register your website with the[ReCaptcha service](https://www.google.com/recaptcha/about/). When you've completed registration, you'll get a public key and a private key.
```

Comment:
I suggest a slightly different link. This one is referenced on https://developers.google.com/recaptcha/intro, where it specifically talks about signing up for the API key pair:
```suggestion
1. Register your website with the [ReCaptcha service](http://www.google.com/recaptcha/admin). When you've completed registration, you'll get a public key and a private key.
```